### PR TITLE
Add video page (Fixes #1418)

### LINKED ISSUE #1418: Design the Videos Page (and add more links)
Update https://github.com/babel/website/blob/master/docs/community/videos.md

Other vids?

- https://www.youtube.com/watch?v=VBscbcm2Mok&index=3&list=PLV5CVI1eNcJgNqzNwcs4UKrlJdhfDjshf
- https://youtu.be/rpb9iOKyO30
- https://www.youtube.com/watch?v=wo8ZLdveIcI&t=10s&list=PLtAjOQfgHj2W7vDBcfcM99ZCUm39jkefR&index=7

### PR DESCRIPTION
**Pull request by zsoltime**
I've just created a new video page (issue #1418). I created a `video.html` in the root directory, but kept the permalink as `/docs/community/videos/`. Not sure if it was the best way. Any opinion on this?

Currently, `_data/videos.yml` contains the video title, author, event, embed link and a short description. Should it contain anything else? Like year, author link, event link?

Happy to make changes if required 😄

### CODE REVIEW

**Comment by existentialism** [comment]
@zsoltime nice!

I think we can drop the descriptions (like: "Henry Zhu explains Babel at React Rally EU."), seem redundant to me?

File: _data/videos.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- category: Meetups
+  items:
+    - title: Babel Sucks
+      author: James Kyle
+      event: Modern Web UI
+      link: https://www.youtube.com/embed/m-HjKwyQoQg
+      description: James Kyle talks Babel at Modern Web UI.
+    - title: Babel - Beyond the Basics (March 2015)
+      author: Sebastian McKenzie
+      event: MelbJS.......
```
Comment (existentialism) [inline _data/videos.yml]: Trailing periods intended/needed?

Comment (zsoltime) [reply _data/videos.yml]: Good catch, not intended, must have been my ring finger :)

**Comment by zsoltime** [comment]
Thank you for the quick review, @existentialism. Yes, it seemed to be redundant, I wasn't sure if we could come up with a different description. I'll remove it shortly. 

Previously @hzoo added author links and event links to the videos. Should I add them, too?

**Comment by hzoo** [comment]
Yeah I think a description is good if it actually explains the content otherwise yeah we should remove it.

Author/event links sound good if not too much work! Looks good

**Comment by zsoltime** [comment]
Not at all. I've just pushed a commit with the links added.

**Comment by hzoo** [comment]
looks good! not sure why one of the meetup videos is too large though

**Comment by existentialism** [comment]
Yeah, I think it'd be okay if they were all the same width (`flex: 0 1 auto`).

Otherwise, LGTM 👍

**Comment by zsoltime** [comment]
@hzoo, it's because of the flexbox settings; if there is a single element in the last row it takes up full width. I'm changing it right now ;)

**Comment by existentialism** [comment]
@zsoltime great stuff, thanks!

**Pull request merged**